OpenSim/Simulation/Test/testContactGeometry.cpp

```diff
@@ -380,80 +380,101 @@ void compareHertzAndMeshContactResults()
 
 // In version 4.0, we introduced intermediate PhysicalFrames to
 // ContactGeometry. The test below ensures that the intermediate frames (as
-// well as the ContactGeometry's location property) are acccounted for by
-// comparing results for equivalent systems, some of which use an intermediate
-// frame.
-// The system is a point mass situated 1 meter along a massless bar that is
-// attached to ground by a hinge (an inverted pendulum). The contact sphere is
-// 1 meter up and 0.5 meters to the right of the mass.
+// well as the ContactGeometry's location and orientation properties) are
+// acccounted for by comparing results for equivalent systems, some of which
+// use an intermediate offset frame.
+// The system is a point mass situated 1 meter along a link that is attached to
+// ground by a hinge. The contact ball (sphere or mesh) is 1 meter up and 0.5
+// meters to the right of the mass (in the frame of the link). The ball bounces
+// on a platform, whose orientation is horizontal (or, 90 degrees clockwise
+// from its "default" orientation, which is vertical).
 //
-//                                      0.5 m
-//                                     -----(geom)
-//                                     |
-//                                     |
-//                                     | 1 m
-//                             1 m     |
+//                                       0.5 m
+//                    .                -----(ball)
+//                    .   (platform)   |
+//                    . 90deg.-`       |
+//                    .   .-`          | 1 m
+//                    ..-`     1 m     |
 //                  (hinge)----------(mass)
-// 
-// The pendulum starts at 27 degrees and then the bar drops down and hits the
-// ground.
+//                        
```

Comment:
The 45 degree line is confusing especially with the `90deg` annotation. Otherwise, your description is clear. You could try to draw the plane.